### ISSUE TITLE
[HW3] user 정보 처리 관련 질문

### ISSUE THREAD

**Issue by rosyrosie** [swsnu/swppfall2022#78]
안녕하세요 조교님
과제 스펙 상 로그인이 가능한 유저는 ```db.json```에서 주어진 이메일, 비밀번호와 일치하는 ```id``` 1번의 유저밖에 없는 것으로 이해하였는데,
이에 따라 로그인 후 ```user```의 정보를 추가적인 절차 없이 ```{id: 1, email: swpp@snu.ac.kr, password: iluvswpp, name: Software Lover}```로 가정하고 사용해도 될지 궁금합니다.
감사합니다.

**Comment by ccdc-git** [comment]
네, 과제 spec에 나온대로 `email: swpp@snu.ac.kr, password: iluvswpp`에 대해 로그인이 성공하면 됩니다.
